app/components/sideBar.module.css

```
@@ -0,0 +1,17 @@
+.sideBarContainer {
+  margin-right: 5%;
+  position: sticky;
+  top: 0;
+  height: fit-content;
+}
+
+.sideBar {
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+  gap: 0.5rem;
+  .link {
+    font-size: 1.2em;
```

```suggestion
  margin-top: 32px;
  gap: 8px;
  .link {
    font-size: 1.2rem;
```
余白の寸法には px を, フォントサイズには rem を使いたいです. こうしておくことで, ブラウザのフォントサイズ設定を尊重しつつ, ボタン間の間隔などは維持できます.